templates/addons/calico.yaml

```diff
@@ -0,0 +1,787 @@
+---
+# Source: calico/templates/calico-config.yaml
```

Comment:
this template is based on https://docs.projectcalico.org/v3.12/manifests/calico.yaml, with minimal changes